cfme/base/ui.py

```diff
@@ -470,7 +472,9 @@ class Chargeback(CFMENavigateStep):
     prerequisite = NavigateToSibling('LoggedIn')
 
     def step(self, *args, **kwargs):
-        self.prerequisite_view.navigation.select('Cloud Intel', 'Chargeback')
+        self.prerequisite_view.navigation.select(
+            self.view.context["object"].intel_name, "Chargeback"
```

Comment:
Within navigation destinations, you can/should reference the registered object via `self.obj`